### PR TITLE
fix person parsing from justice.cz and registration date and person t…

### PR DESCRIPTION
**Pull request by radoslavius**
…ype fields added

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Pull request does not pass validation.

**Review by ghost — CHANGES_REQUESTED** [review]
Pull request does not pass validation.

**Review by ghost — CHANGES_REQUESTED** [review]
Pull request does not pass validation.

**Review by ghost — CHANGES_REQUESTED** [review]
Pull request does not pass validation.

**Review by ghost — CHANGES_REQUESTED** [review]
Pull request does not pass validation.

**Review by ghost — CHANGES_REQUESTED** [review]
Pull request does not pass validation.

File: src/Parser/PersonParser.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    $name = trim(explode(', dat. nar.', $contentItems[0])[0]);
+
+                    try {
+                        $birthday = DateTimeParser::parseFromCzechDateString($contentItems[1]);
+                        $address = $contentItems[2];
+                    } catch (\Exception $e) {
+                        $birthday = null;
+                    }
+
+                    return ['name' => ucwords(mb_strtolower($name)), 'birthday' => $birthday, 'address' => isset($address) ? $address : ''];
```
Comment (filipmelik) [inline src/Parser/PersonParser.php:44]: I have found that `ucwords` does not handle words starting with diacritics properly. Let me suggest a fix I did in https://github.com/filipmelik/Ares/commit/c692788fa98cd94af4d5664f08fc3e04de8f121f (along with some code style changes)

**Comment by dfridrich** [comment]
See #33

**Pull request closed without merging**